libminifi/test/sensors-tests/SensorTests.cpp

```diff
@@ -78,7 +78,7 @@ class PcapTestHarness : public IntegrationBase {
   }
 
  protected:
-  char *dir;
+  std::string dir;
```

Comment:
On line 50, `createTempDirectory` returns a reference to the local `format` buffer, making this member dangling after construction in the old version.